solarwinds_apm/apm_config.py

```diff
@@ -135,117 +115,10 @@ def __init__(
         )
 
         # Update and apply logging settings to Python logger
-        self.update_log_settings()
-        apm_logging.set_sw_log_type(
-            self.__config["log_type"],
-            self.__config["log_filepath"],
-        )
         apm_logging.set_sw_log_level(self.__config["debug_level"])
 
-        self.update_log_filepath_for_reporter()
-
-        # Calculate c-lib extension usage
-        # (
-        #     self.extension,
-        #     self.context,
-        #     oboe_api_swig,
-        #     oboe_api_options_swig,
-        # ) = self._get_extension_components(
-        #     self.agent_enabled,
-        #     self.is_lambda,
-        # )
-
-        # Create OboeAPI options using extension and __config
-        # oboe_api_options = oboe_api_options_swig()
-        # oboe_api_options.logging_options.level = self.__config["debug_level"]
-        # oboe_api_options.logging_options.type = self.__config["log_type"]
-        # self.oboe_api = oboe_api_swig(
-        #     oboe_api_options,
-        # )
-        #
-        # self.context.setTracingMode(self.__config["tracing_mode"])
-        # self.context.setTriggerMode(self.__config["trigger_trace"])
-        #
-        # # (Re-)Calculate config if AppOptics
-        # self.metric_format = self._calculate_metric_format()
-        # self.certificates = self._calculate_certificates()
-        # self.__config["export_logs_enabled"] = self._calculate_logs_enabled()
-        # self.__config["export_metrics_enabled"] = (
-        #     self._calculate_metrics_enabled()
-        # )
-
         logger.debug("Set ApmConfig as: %s", self)
 
-    # def _get_extension_components(
```

Comment:
I agree with completely removing this helper and its callers now, since your Django test result looks good (HttpSampler) as well as the test Lambda layer I created (JsonSampler)(see ticket).